node/core/approval-voting/src/tests.rs

```diff
@@ -513,9 +658,64 @@ fn blank_subsystem_act_on_bad_block() {
 }
 
 #[test]
-fn ss_rejects_approval_if_no_block_entry() {
+fn subsystem_rejects_approval_if_no_candidate_entry() {
+	let store = SharedTestStore::new(TestStore::default());
+	let (oracle, _handle) = make_sync_oracle(false);
+	let (mock_clock, assignment_criteria) = make_harness_input();
```

Comment:
nit: this is rather repetitive, a builder pattern with replacable defaults comes to mind that would reduce this a bit, but hey, it's a test, so feel free to ignore.